mongorm/fields/SafeDictField.py

```diff
@@ -20,12 +23,12 @@ def deepCoded( dictionary, coder ):
 	return dictionary
 
 def encode( string ):
-	if isinstance(string, unicode):
+	if isinstance(string, str):
 		string = string.encode( 'utf-8' )
-	return string.encode( 'hex' )
+	return bytes(binascii.hexlify(string)).decode('utf-8')
```

Comment:
I wasn't sure whether you need the `bytes()` here? When I tried out `binascii.hexlify` it accepts only a byte object and returns a byte object.

Reply:
This was a tough section to get working properly, but I eventually went with this route so that it would be the exact same line that worked in python 2/3. The cast to bytes here is so that it's a `builtins.bytes` instead of a py2 bytes. It's not a massive thing, but it better ensures that it's actually a usable bytes object

Reply:
Ah ok make sense. 👍